src/rustdoc.md

```diff
@@ -4,7 +4,7 @@ Rustdoc actually uses the rustc internals directly. It lives in-tree with the
 compiler and standard library. This chapter is about how it works. (A new
 implementation is also [under way], though).
```

Comment:
Could you change this to something more vague (as per https://github.com/rust-lang-nursery/rustc-guide/pull/150#issuecomment-395783504)

Perhaps something like "There have been some attempts to rewrite rustdoc, but they are in early stages".